sources/getting-started/functional-api-guide.md

```diff
@@ -293,7 +293,7 @@ assert conv.get_input_shape_at(1) == (None, 64, 64, 3)
 
 ## More examples
 
-コード例を見ることは訓練時に非常に有効です．
+コード例から学び始めることは最良の手法です。
```

Comment:
Please replace `。with ` to `．`.

Reply:
Thank you for pointing it out.
I have fixed the punctuation issue and committed it.

Reply:
Thank you!